### DISCUSSION TITLE
Should the `rename-project` script delete the `tmp` directory?

### DISCUSSION THREAD

**Discussion by edruder** [nickjj/docker-rails-example#75] [Ideas]
I was bit by the `RuntimeError: invalid bytecode` problem after renaming the project while I was doing some initial playing around with this project. The note in the README about this error and how to resolve it (`./run clean` or delete the `/tmp` directory) didn't jump out at me, causing me some frustration.

Wouldn't it make sense to have the `bin/rename-project` script _always_ delete the `/tmp` directory? Rails will always recreate it when it needs it, apparently.

**Comment by nickjj** [comment]
That's a good idea.

I'd suggest recursively deleting all of the contents in the tmp directory except for the `.keep` file since `tmp/.keep` is committed to version control. This will leave the original modified times on the directory and keep file too.

Do you want to make a PR for that so you get credit or do you want me to make the change? If you do make that PR you can delete the message in the README file that you're referencing since it's no longer needed.

**Reply by edruder** [reply]
I'll put together a PR to do it!

**Reply by edruder** [reply]
#76
